isb_lib/opencontext_adapter.py

```diff
@@ -161,16 +160,18 @@ def _getPage(self):
     def last_url_str(self) -> typing.AnyStr:
         return self.url
 
+
 def _validate_resolved_content(thing: isb_lib.models.thing.Thing):
     isb_lib.core.validate_resolved_content(OpenContextItem.AUTHORITY_ID, thing)
 
+
 def reparse_as_core_record(thing: isb_lib.models.thing.Thing) -> typing.List[typing.Dict]:
     _validate_resolved_content(thing)
     try:
         transformer = isamples_metadata.OpenContextTransformer.OpenContextTransformer(thing.resolved_content)
         return [isb_lib.core.coreRecordAsSolrDoc(transformer)]
     except Exception as e:
-        get_logger().error("Failed trying to run transformer on %s", str(thing.resolved_content))
+        get_logger().error(f"Failed trying to run transformer on {str(thing.resolved_content)}, exception: {e}")
```

Comment:
I think % formatting should be used in log messages as opposed to fstrings because the fstrings are computed even if the log level is insufficient to emit the message, whereas % formatting is not.

Also, I don't thing the str() is needed as the logger should do that - if the message is to be emitted

Reply:
Good point on the log messages, I had forgotten that.

Reply:
And I wrote a test to make sure that the formatting works as expected:

```
dict = {
    "foo": "bar"
}
logging.error("This is a dict error: %s", dict)

try:
    raise AttributeError("This is an attribute error")
except Exception as e:
    logging.error("The exception was %s", e)
```

yields:

```
ERROR:root:This is a dict error: {'foo': 'bar'}
ERROR:root:The exception was This is an attribute error
```